Tests/UI/3DS/ThreedsWebViewControllerTests.swift

```diff
@@ -122,11 +122,6 @@ class ThreedsWebViewControllerTests: XCTestCase {
         _ = ThreedsWebViewController(nibName: nil, bundle: nil)
     }
 
-    func testInitializationCoder() {
-        let coder = NSKeyedUnarchiver(forReadingWith: Data())
-        _ = ThreedsWebViewController(coder: coder)
-    }
```

Comment:
Syntax changed for `NSKeyedUnarchiver` and its behaviour no longer allows it to create with empty data, throwing an error.

Reply:
just an uneducated guess: does it accept if we create a dummy data from a dummy string?

Reply:
Honestly I'm not sure. Don't know why it was tested either, as init with coder as far as I know is triggered when loading via Storyboard 🤔, so the test didn't really make sense to begin with 🤷‍♂️.

Reply:
I'd personally hesitate to remove a test because there might be a scenario that's hard to catch and that might've been handling that for a while but still don't mind it's getting deleted if you're confident :)